src/StakeRegistry.sol

```diff
@@ -78,9 +56,14 @@ contract StakeRegistry is VoteWeigherBase, StakeRegistryStorage {
         // for each quorum, loop through operators and see if they are a part of the quorum
         // if they are, get their new weight and update their individual stake history and the
         // quorum's total stake history accordingly
+        uint8 quorumCount = registryCoordinator.quorumCount();
         for (uint8 quorumNumber = 0; quorumNumber < quorumCount; ) {
             int256 totalStakeDelta;
 
+            // TODO - not a huge fan of this dependency on the reg coord, but i do prefer this
+            //        over the stakereg also keeping its own count.
```

Comment:
I think this makes sense. it's a bit ugly, but in my mind the registries are basically all "puppets" of the RegistryCoordinator, so calling into the coordinator to fetch a bit of data is fine.
I wouldn't be super opposed to just keeping count in the StakeRegistry either, as it'd be more efficient -- we could just increment a `quorumCount` variable in the `initializeQuorum()` function? Then the two can only be out-of-sync if the RegistryCoordinator either fails to call `initializeQuorum` appropriately or calls it inappropriately.

Reply:
This is marked TODO because I was intending a future PR to move `updateStakes` into the registry coordinator. I'm working on that locally and will have something to share next week

Reply:
ahhh OK that works!